src/app/view/components/listing-data-table/listing-data-table.components.spec.ts

```diff
@@ -80,10 +86,17 @@ describe('Listing Component', () => {
         columns: new SimpleChange(null, comp.columns, true)
     });
 
+    let dataList = fixture.debugElement.query(By.css('p-dataTable'));
```

Comment:
this styles could be good to encapsulate in a interface or class or constants. When you need to change any of them it would be easier than search/replace